src/main/java/duke/UI.java

```diff
@@ -0,0 +1,142 @@
+package duke;
+
+import duke.task.Task;
+
+import java.util.ArrayList;
+
+public class UI{
+    private static final String BUFFER_LINE = "____________________________________________________________\n";
+
+    public static void printWelcomeMessage() {
+        System.out.println(BUFFER_LINE);
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        String greeting = BUFFER_LINE
+                + " Hello! I'm Duke\n"
+                + " What can I do for you?\n"
+                + BUFFER_LINE;
+        System.out.println(greeting);
+    }
+
+    public static void printEndMessage() {
+        String byeMessage = BUFFER_LINE
+                + " Bye. Hope to see you again soon!\n"
+                + BUFFER_LINE;
+        System.out.println(byeMessage);
+    }
+
+    public static void printInvalidMessage() {
+        String invalidInput = BUFFER_LINE
+                + " Oops! Looks like I can't read that! Please input a valid command.\n"
+                + BUFFER_LINE;
+        System.out.println(invalidInput);
+    }
+
+    public static void printAdditionMessage(Task current, int taskCount) {
+        String addition = BUFFER_LINE
+                + " Gotcha! I've added this task: \n"
+                + "    " + current.listTask() + "\n"
+                + " Now you have " + taskCount + " tasks in the list.\n"
+                + BUFFER_LINE;
+        System.out.println(addition);
+    }
+
+    public static void printDoneMessage(Task current) {
+        String doneMessage = BUFFER_LINE
+                + " Nice! I've marked this task as done: \n"
+                + "    " + current.listTask() + "\n"
+                + BUFFER_LINE;
+        System.out.println(doneMessage);
+    }
+
+    public static void printDeleteMessage(Task current, int taskCount) {
+        String doneMessage = BUFFER_LINE
+                + " Alright! I've removed this task from the list: \n"
+                + "    " + current.listTask() + "\n"
+                + " Now you have " + taskCount + " tasks in the list.\n"
+                + BUFFER_LINE;
+        System.out.println(doneMessage);
+    }
+
+    public static void printList(ArrayList<Task> tasks) {
+        System.out.println(BUFFER_LINE);
+        System.out.println(" Here are the tasks in your list:");
+        int i = 1;
+        for (Task task : tasks) {
+            if (task == null) {
+                break;
+            }
+            System.out.println(" " + i + ". " + task.listTask());
+            i = i + 1;
+        }
+        System.out.println(BUFFER_LINE);
+    }
+
+    public static void printEmptyListMessage() {
+        String emptyListMessage = BUFFER_LINE
```

Comment:
If the string is a constant, the variable name must be all uppercase (and underscore to separate the words) instead.